### PR TITLE
Update macOS steps to fix Homebrew information and correct syntax

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description

A review of macOS installation steps showed that the install information for Agent static install was incorrect in some areas and other important information was completely missing.

1. The default prefix for Homebrew is based on CPU type (Intel vs Apple Silicon). Added a note about this.
2. The information about where grafana-agent is installed by default is now based on the CPU type.
3. Removed `mkdir $(brew --prefix)/etc/grafana-agent/` because this directory is created by the installer.
4. The sections are now split by task with new headings for each task.
5. Clarified the information around where the `config.yml` file is located - this was confusing people.
6. The log information is updated to match the Flow style.

#### PR Checklist

- [ ] CHANGELOG updated
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Comment by clayton-cornell** [comment]
Test build to use for review

image

File: docs/sources/static/set-up/install-agent-macos.md [hunk 1]
```diff
@@ -11,46 +11,58 @@ You can install Grafana Agent in static mode on macOS.
...
+* `$(brew --prefix)/var/log/grafana-agent.log`
+* `$(brew --prefix)/var/log/grafana-agent.err.log`
+
+## Upgrading Grafana Agent
+
+Open a terminal and run the following command to upgrade Grafana Agent:
+
+```shell
+brew upgrade grafana-agent
+ ```
```
Comment (rfratto) [inline docs/sources/static/set-up/install-agent-macos.md:68]: In my experience, you have to restart the service after upgrading, making the instructions:

```shell
brew upgrade grafana-agent 
brew services restart grafana-agent
```

**Review by rfratto — APPROVED** [review]
LGTM % one piece of feedback about upgrading

File: docs/sources/static/set-up/install-agent-macos.md [hunk 2]
```diff
@@ -11,46 +11,58 @@ You can install Grafana Agent in static mode on macOS.
...
-{{%/admonition %}}
+* `$(brew --prefix)/var/log/grafana-agent.log`
+* `$(brew --prefix)/var/log/grafana-agent.err.log`
+
+## Upgrading Grafana Agent
+
+Open a terminal and run the following command to upgrade Grafana Agent:
+
+```shell
+brew upgrade grafana-agent
```
Comment (clayton-cornell) [inline docs/sources/static/set-up/install-agent-macos.md:67]:
```suggestion
brew upgrade grafana-agent
brew services restart grafana-agent
```

File: docs/sources/static/set-up/install-agent-macos.md [hunk 3]
```diff
@@ -11,46 +11,58 @@ You can install Grafana Agent in static mode on macOS.
...
 
-{{% admonition type="note" %}}
-To send your data to Grafana Cloud, set up Grafana Agent using the Grafana Cloud integration. Refer to [how to install an integration](/docs/grafana-cloud/data-configuration/integrations/install-and-manage-integrations/) and [macOS integration](/docs/grafana-cloud/data-configuration/integrations/integration-reference/integration-macos-node/).
-{{%/admonition %}}
+* `$(brew --prefix)/var/log/grafana-agent.log`
+* `$(brew --prefix)/var/log/grafana-agent.err.log`
+
+## Upgrading Grafana Agent
+
+Open a terminal and run the following command to upgrade Grafana Agent:
```
Comment (clayton-cornell) [inline docs/sources/static/set-up/install-agent-macos.md:64]:
```suggestion
Open a terminal and run the following commands to upgrade and restart Grafana Agent:
```

File: docs/sources/static/set-up/install-agent-macos.md (hunk 1 above)
Comment (clayton-cornell) [reply docs/sources/static/set-up/install-agent-macos.md]: Updated the step

**Pull request merged**